Airports/EGHO/Positions.txt

```diff
@@ -0,0 +1 @@
+EGHO_R_TWR:Thruxton Radio:120.250:HOT:S:EGHO:TWR:-:-:::N051.12.38.000:W001.36.00.000::
```

Comment:
In the AIP @ `EGHO AD 2.18`, the A/Gs frequency is 118.280 - and it doesn't conflict with any other frequency as per the .ese included in our current SF.
![image](https://user-images.githubusercontent.com/69194446/160298811-771860d8-e2aa-4b49-8aa9-c64fa5ea0702.png)
```suggestion
EGHO_R_TWR:Thruxton Radio:118.280:HOT:S:EGHO:TWR:-:-:::N051.12.38.000:W001.36.00.000::
```

Reply:
VATSIM does not support 8.33kHz frequencies, but closest is 118.275 per the Basic.txt, I've updated it to that. Just forgot to change it when I copied from EGCB, good spot thanks.

Peter